account_financial_report_qweb/view/report_template.xml

```diff
@@ -4,7 +4,6 @@
     <template id="account_financial_report_assets_backend"
               name="account_financial_report assets" inherit_id="web.assets_backend">
         <xpath expr="." position="inside">
-            <link href="/account_financial_report_qweb/static/src/css/report.css" rel="stylesheet"/>
```

Comment:
AFAICS ths file is not being used anymore, so you could remove it, right?

Or, if it is still needed for reports, then move it to some `assets_report` view.

Reply:
It is used here: https://github.com/OCA/account-financial-reporting/blob/25e705cf833ac68bc3630d69ca2934f1e08de987/account_financial_report_qweb/report/templates/layouts.xml#L4-L8